<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextInput</class>
 <widget class="QDialog" name="TextInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FlrmapBar { background-color: rgba(255, 255, 255, 0); }

QLabel#label_tip { font: bold &quot;Arial&quot;; font-size:16pt; color: rgba(0,160,233,255); }

QLineEdit{ border-image: url(:/rc/15_Flrmap/edit_num.png); background-color: rgba(255, 255, 255, 0); font: large &quot;Arial&quot;; font-size:22pt; color: rgba(255,255,255,250); border:0px solid rgb(10,10,10);/*border-radius:15px;*/}

QPushButton#btn_ok{ border-image: url(:/0_common/rc/0_common/btnadd.png);  font: bold &quot;Arial&quot;; font-size:18pt; color: rgba(255,255,255,255);}
QPushButton#btn_ok::pressed{ border-image: url(:/0_common/rc/0_common/btnlight2.png); }

QPushButton#btn_cancel{ border-image: url(:/0_common/rc/0_common/btndel.png);  font: bold &quot;Arial&quot;; font-size:18pt; color: rgba(255,255,255,255);}
QPushButton#btn_cancel::pressed{ border-image: url(:/0_common/rc/0_common/btnlight2.png); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="LineEdit" name="edit_text">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">bstui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
